<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Mouth Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="eventListWidget">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showEventChkBox">
          <property name="text">
           <string>Show all events</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="processXMLChkBox">
          <property name="text">
           <string>Process XML</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="speak_text">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="VoiceLbl">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Voice</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="VoiceCombox">
              <item>
               <property name="text">
                <string>中国国语</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>中国粤语</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>英式英语</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>美式英语</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="RateLbl">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="RateHSdr">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="VolumeLbl">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Volume</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="VolumeHSdr">
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="PropertyLbl">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Propertiy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="ageSpinBox"/>
            </item>
            <item>
             <widget class="QComboBox" name="genderComBox">
              <item>
               <property name="text">
                <string>男</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>女</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="openFileBtn">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Open File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="playBtn">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Speak</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pauseBtn">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Pause</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopBtn">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="skipBtn">
                 <property name="text">
                  <string>skip</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="skipSpinBox"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="speakWavBtn">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Speak .wav</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetBtn">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveToFileBtn">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>Save to .wav</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="aboutBtn">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>28</height>
              </size>
             </property>
             <property name="text">
              <string>About</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
